src/js/components/StatusBar.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react';
+import { View, StyleSheet, StatusBar } from 'react-native';
+import { IS_IOS, IS_IPHONEX } from '../config/Constants';
+import { colors } from '../utils/colors';
+
+export const STATUS_BAR_HEIGHT = IS_IOS ? (IS_IPHONEX ? 44 : 20) : (StatusBar.currentHeight || 0);
```

Comment:
Can the logic behind this please be explained in a code comment? I assume it has to do with the notch on iPhone X

Reply:
Yes, added a comment